atest/robot/output/listener_interface/unsupported_listener_version.robot

```diff
@@ -10,10 +10,6 @@ Unsupported version
     1    unsupported_listeners.InvalidVersionClassListener
     ...    Listener 'unsupported_listeners.InvalidVersionClassListener' uses unsupported API version 'kekkonen'.
 
-No version information
-    2    unsupported_listeners
-    ...    Listener 'unsupported_listeners' does not have mandatory 'ROBOT_LISTENER_API_VERSION' attribute.
-
```

Comment:
A new test is needed to make sure using a listener without `ROBOT_LISTENER_API_VERSION` is possible.